lms/templates/goal_reminder_banner.html

```diff
@@ -1,59 +1,18 @@
-{% load i18n %}
 {# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
-{# we're using important below to override inline styles and my understanding is for email clients where media queries do not work, they'll simply see the desktop css on their phone #}
 <style>
-.goals-two-color:first-line { color: white !important;}
-.goals-two-color {
-    font-size: 2.686rem;
-    line-height: 3rem;
-    margin: 4.5rem 0 3.44rem 1.0625rem;
-    padding-left: 1.25rem;
-    width: 18.75rem;
-    top: 100px;
-}
-@media only screen and (max-width: 575px) {
-  .goals-two-color {
-    font-size: 2.2rem;
-    line-height: 2rem;
-    width: 15rem;
-  }
-}
 @media only screen and (max-width: 456px) {
-  .goals-two-color {
-    font-size: 1.8rem;
-    line-height: 1.8rem;
-    width: 12.5rem;
-    margin: 2.25rem 0;
-    top: 7.5rem;
-  }
-}
-@media only screen and (max-width: 375px) {
-  .goals-two-color {
-    font-size: 1.35rem;
-    line-height: 1.65rem;
-    top: 6.875rem;
-  }
-}
-@media only screen and (max-width: 325px) {
-  .goals-two-color {
-    top: 6.25rem;
+  .goals-banner-div {
+    background-image: url('{{image_url}}goalreminder-mobile-hero.png');
   }
 }
 </style>
-<div class="goals-banner-wrapper-div" style="width:inherit; font-family: Inter, Arial, sans-serif;">
-    <div class="goals-banner-div" style="width: inherit; margin-left: -1.25rem; margin-top: -1rem;">
-      <img src="{{image_url}}images/goalreminder-desktop-hero.png" style="
-          width: calc(100% + 20px);
-          height: auto;
-      ">
-        <h2 class="goals-two-color" style="
-          color: #F0CD01;
-          font-weight: bold;
-          position: absolute;
-        ">
-            {% filter force_escape %}{% blocktrans %}
-                You’re almost there!
-            {% endblocktrans %}{% endfilter %}
-        </h2>
-    </div>
-</div>
\ No newline at end of file
+<div class="goals-banner-div" style="
+    margin-left: -1.25rem;
+    margin-top: -1rem;
+    width: calc(100% + 20px);
+    height: 0;
+    padding-top: 41%;
```

Comment:
this is an odd hack i got from here to scale the image to the size of the container while preserving aspect ratio - not sure if there is a cleaner way to do this
however, I don't think I could simply use an image tag because I'm using media queries above to try to sub in a different png for mobile (and as I had previously mentioned svg doesn't work for email)